utils/eventtracking.js

```diff
@@ -109,5 +109,7 @@ module.exports = {
     EVENT_ID_REPL_END: 'shell_repl_end',
     EVENT_ID_GENERATE_KEY_START: 'shell_generate_key_start',
     EVENT_ID_GENERATE_KEY_END: 'shell_id_generate_key_end',
+    EVENT_ID_DELETE_KEY_START: 'event_id_delete_key_start',
```

Comment:
@mattlockyer not really comment on this PR, but relevant context – what do you think on https://github.com/near/near-shell/issues/306#issuecomment-630612739

Should we make events automatic for commands?

Reply:
We're trying to capture the best metrics. We won't get it right the first time. We should be opt-in and collect everything we can, unless the user explicitly opts out.
Therefore a whitelist for events seems like an anti-pattern and all events should get logged automatically with a blacklist (if things are too noisy). But noise is also something you can just filter out later in your analytics.

Reply:
just realized I wanted to tag @mikedotexe (as it's his PR), but thanks for input! I agree that logging by default is good idea for near-shell commands

Reply:
@vgrichina I think I'll answer over there in the issue. I really don't like the START and END, but we haven't yet implemented crash reporting so doing what we have. I think the more events we track the better. Thanks for tagging that other issue, I'll provide feedback